integration-tests/Cargo.toml

```diff
@@ -10,13 +10,13 @@ repository = "https://github.com/galacticcouncil/Basilisk-node"
 
 [dependencies]
 # Warehouse dependencies
-pallet-transaction-multi-payment = { git = "https://github.com/galacticcouncil/warehouse", rev = "e85c3c68708c7d9789d18c0a230efd436fc5ea10", default-features = false}
-pallet-price-oracle = { git = "https://github.com/galacticcouncil/warehouse", rev = "e85c3c68708c7d9789d18c0a230efd436fc5ea10", default-features = false}
-pallet-nft = { git = "https://github.com/galacticcouncil/warehouse", rev = "e85c3c68708c7d9789d18c0a230efd436fc5ea10", default-features = false }
-pallet-currencies = { git = "https://github.com/galacticcouncil/warehouse", rev = "e85c3c68708c7d9789d18c0a230efd436fc5ea10", default-features = false }
-pallet-route-executor = { git = "https://github.com/galacticcouncil/warehouse", rev = "e85c3c68708c7d9789d18c0a230efd436fc5ea10", default-features = false }
+pallet-transaction-multi-payment = { git = "https://github.com/galacticcouncil/warehouse", rev = "a7e28a3a7fba655d9d27668097a1666f4250a495", default-features = false}
```

Comment:
latest commit
```suggestion
pallet-transaction-multi-payment = { git = "https://github.com/galacticcouncil/warehouse", rev = "4223029df725ee9377d86fa5e02b220c59619cf6", default-features = false}
```

Reply:
you can use this command to replace it everywhere `sed -i '' 's/e85c3c68708c7d9789d18c0a230efd436fc5ea10/4223029df725ee9377d86fa5e02b220c59619cf6/g' ./**/*.toml`